### DISCUSSION TITLE
Finding release catno

### DISCUSSION THREAD

**Discussion by masquinongy** [joalla/discogs_client#37] [Support] [answered]
Hey, how do I get the catalog number for a release? Thanks.

**Comment by VinylVault** [comment, marked as answer]
@masquinongy the catalogue number is listed under the company info, same record set as the Record Label information ... `company.data['catno']` is where I get it from

**Reply by alifhughes** [reply]
Thanks for answering @VinylVault, you are correct. You can access the `catno` for a `company` or a `label` by doing:
```python
company.data["catno"]
label.data["catno"]
```
This can be seen in the release response [here](https://www.discogs.com/developers#page:database,header:database-release). Unfortantely it seems you can't get the `catno` for a `release` itself mind.

**Reply by Simon-Dixon** [reply]
I just had the same problem.  But I think the reason for a `release` not having a `catno` is that some releases have multiple label/catno combinations, so you have to specify which one you want  e.g. via your choice of `n` in   `myrelease.labels[n].fetch("catno")`
